docs/concepts/backup-restore.md

```diff
@@ -0,0 +1,39 @@
+# Backup and restore
+
+Kubernetes uses Etcd as the key-value store for its resource definitions. Gardener supports the backup and restore of etcd. It is the reponsibility of the shoot owners to backup the workload data.
+
+
+Gardener uses [etcd-backup-restore] component to backup the etcd backing the Shoot cluster regularly and restore in case of disaster. It is deployed as sidecar via [etcd-druid]. This doc mainly focuses on the backup and restore configuration used by gardener when deploying these components. For more details on the design and internal implementation details, please refer [GEP-06](../06-etcd-druid.md) and documentation on individual repository.
```

Comment:
```suggestion
Gardener uses [etcd-backup-restore](<link>) component to backup the etcd backing the Shoot cluster regularly and restore in case of disaster. It is deployed as sidecar via [etcd-druid](<link>). This doc mainly focuses on the backup and restore configuration used by Gardener when deploying these components. For more details on the design and internal implementation details, please refer [GEP-06](../../proposals/06-etcd-druid.md) and documentation on individual repository.
```